pkg/scheduler/apis/config/types_pluginargs.go

```diff
@@ -218,3 +221,34 @@ type NodeAffinityArgs struct {
 	// a specific Node (such as Daemonset Pods) might remain unschedulable.
 	AddedAffinity *v1.NodeAffinity
 }
+
+type StrategyType string
+
+const (
+	LeastAllocated           StrategyType = "LeastAllocated"
+	MostAllocated            StrategyType = "MostAllocated"
+	RequestedToCapacityRatio StrategyType = "RequestedToCapacityRatio"
+)
+
+// ScoringStrategy define StrategyType for node resource plugin
+type ScoringStrategy struct {
+	metav1.TypeMeta
+
+	// Strategy selects which strategy to run.
+	Strategy StrategyType
+
+	// Resources to consider when scoring.
+	// The default resource set includes "cpu" and "memory" with an equal weight.
+	// Allowed weights go from 1 to 100.
+	Resources []ResourceSpec
+
+	// Arguments specific to RequestedToCapacityRatio strategy.
+	RequestedToCapacityRatio *RequestedToCapacityRatioArg
+}
+
+// RequestedToCapacityRatioArg define RequestedToCapacityRatio args
+// TODO: RequestedToCapacityRatioArg should rename to RequestedToCapacityRatioArgs when above duplicate struct removed
+type RequestedToCapacityRatioArg struct {
```

Comment:
lets name it `RequestedToCapacityRatioParam` since Args is reserved for higher level